source/includes/api/_widget.md

```diff
@@ -1,428 +1,296 @@
-
-<aside class="notice">
-   Using `userId` filter will return widgets created/cloned by the user. This filter is not compatible with `favourite` or `collection`.
-   If you want to fetch favourited widgets, or specific ones from a collection, do not include the `userId` param.
-</aside>
-
-> Return the widgets filtered whose name contains glad
+> Custom pagination: load page 2 using 25 results per page
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/widget?name=glad
+curl -X GET "https://api.resourcewatch.org/v1/widget?page[number]=2&page[size]=25"
 ```
 
-> Return the widgets filtered by dataset
+Field        |         Description          |   Type |   Default
+------------ | :--------------------------: | -----: | ----------:
+page[size]   | The number elements per page. Values above 100 are not officially supported. | Number | 10
+page[number] |       The page number        | Number | 1
 
-```shell
-curl -X GET https://api.resourcewatch.org/v1/widget?dataset=d02df2f6-d80c-4274-bb6f-f062061655c4
-curl -X GET https://api.resourcewatch.org/v1/dataset/d02df2f6-d80c-4274-bb6f-f062061655c4/widget
-```
+### Filters
 
-> Sort by name
+> Return the widgets filtered by those whose name contains emissions
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/widget?sort=name
+curl -X GET "https://api.resourcewatch.org/v1/widget?name=emissions"
 ```
 
-> Filter widgets by published status
+> Return the widgets filtered by dataset
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/widget?published=false
+curl -X GET "https://api.resourcewatch.org/v1/widget?dataset=11de2bc1-368b-42ed-a207-aaff8ece752b"
+curl -X GET "https://api.resourcewatch.org/v1/dataset/11de2bc1-368b-42ed-a207-aaff8ece752b/widget"
 ```
 
-> Filter widgets by verified status
+> Filter widgets by default value
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/widget?verified=false
+curl -X GET "https://api.resourcewatch.org/v1/widget?default=false"
 ```
 
-> Return the widgets filtered by template
+> Filter widgets by environment
+
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/widget?template=true
+curl -X GET "https://api.resourcewatch.org/v1/widget?env=staging"
 ```
 
-> Filter widgets by default option
+> Return the widgets filtered by those whose applications contain both `rw` and `prep` applications simultaneously
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/widget?default=true
+curl -X GET "https://api.resourcewatch.org/v1/widget?app=rw@prep"
 ```
 
-> Return widgets whose applications contain rw
+The widget list endpoints provide a wide range of filters that you can use to tailor your widget listing. The great majority of the widget fields you'll find on the [widget reference](#widget-reference) section can be used as filters when listing widgets, with the following exceptions:
```

Comment:
Is it actually all of them except these three? If that's the case I would just say "all" rather than "the great majority".